mlir/test/python/ir/array_attributes.py

```diff
@@ -617,3 +618,111 @@ def test_attribute(context, mview):
     # CHECK: BACKING MEMORY DELETED
     # CHECK: EXIT FUNCTION
     print("EXIT FUNCTION")
+
+
+# CHECK-LABEL: TEST: testGetDenseResourceElementsAttrNdarrayI32
+@run
+def testGetDenseResourceElementsAttrNdarrayI32():
+    class DLPackWrapper:
+        def __init__(self, array: np.ndarray):
+            self.dlpack_capsule = array.__dlpack__()
+    
+        def __del__(self):
+            print("BACKING MEMORY DELETED")
```

Comment:
```suggestion
            print("DLPACK MEMORY DELETED")
```
we actually don't own the memory just the metadata about the tensor (dlpack).